third_party/newlib/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+filegroup(
+    name = "newlib_srcs",
+    srcs = glob(["newlib-4.3.0/**"]),
+)
+
+configure_make(
```

Comment:
Was something changed in the `newlib` sources?
And if so, how would we keep updating it to new versions?
Should we try submitting changes to the original repo?

Reply:
Thus far the only change has been adding `oak` to the list of valid operating systems in `config.sub`.

Reply:
Can it be done via Bazel patches?
So that we won't need to add newlib to the third-party, and use Bazel `http_archive` instead?